config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
-  resources :resource_categories, only: [:show, :create, :update, :destroy]
+  resources :resource_categories, only: [:show, :create, :update, :destroy] do
+    resources :resources, only: [:create]
```

Comment:
I'd be interested in hearing feedback on this url schema. Rails style guidelines suggest that you should only nest resources one level deep, so I've opted for something like:
POST `resource-categories/:resource_category_id/resources?state=:state`
as opposed to like `resource-categories/:resource_category_id/states/:state/resources`.

How do people feel about passing in state as a query param?

Reply:
I tend to go for the query params myself, otherwise you wind up with a URL like `resource-categories/:resource_category_id/states/` that doesn't do anything in the middle of your longer structure.

Reply:
Whoops sorry just saw this when @dryan replied. I agree, passing the state in as a query param is preferred :)

Reply:
I also prefer query parameters.